src/rpc/client.cpp

```diff
@@ -315,91 +368,140 @@ static const CRPCConvertParam vRPCConvertParams[] =
     { "stop", 0, "wait" },
     { "addnode", 2, "v2transport" },
     { "addconnection", 2, "v2transport" },
+    { "decodepsbt", 0, "psbt", ParamFormat::STRING },
+    { "analyzepsbt", 0, "psbt", ParamFormat::STRING},
+    { "verifymessage", 1, "signature", ParamFormat::STRING },
+    { "verifymessage", 2, "message", ParamFormat::STRING },
+    { "getnewaddress", 0, "label", ParamFormat::STRING },
+    { "getnewaddress", 1, "address_type", ParamFormat::STRING },
```

Comment:
Should we add other commands which also has the address_type argument to this list?

Passing "=" including param to getnewaddress throw following error:
```
$ build/bin/bitcoin-cli -regtest -named getnewaddress label_string include=equal
error code: -5
error message:
Unknown address type 'include=equal'
```

Passing "=" including param to getrawchangeaddress which has same address_type argument throw following error:
```
$ build/bin/bitcoin-cli -regtest -named getrawchangeaddress include=equal
error code: -8
error message:
Unknown named parameter include
```

It may be good idea to make error consistent for same situation.